### ISSUE TITLE
Membership Enrollment Initiative

### ISSUE THREAD

**Issue by chrisguindon** [EclipseFdn/react-eclipsefdn-members#4]
The goal of this project is to capture the information that we need to facilitate the on-boarding of a new member. That information includes but not limited to: address, logo, and company reps.

We are using https://docs.google.com/document/d/1E1THXZ-YhL_Rp5wMKD79O9Hb1ynQsrlqagnNJE3EPr8/edit# to describe the business requirements.

The first prototype is available via:
https://deploy-preview-1--eclipsefdn-react-members.netlify.app/login

**Comment by shanda-eclipse** [comment]
We are in the process of finalizing the design, but it's 90% of the way there so you should be able to start working on it. 

https://xd.adobe.com/view/583c3ff2-1fdc-43e4-ad8a-6dc730f5bb38-0d6f/

**Comment by flora8984461** [comment]
Rough todo:

- [ ] Company Information
- [ ] Membership Level
- [ ] Working Group
- [ ] Signing Auth
- [ ] Review
- [ ] Log in
- [ ] Style the Steppers
- [ ] What to do with the login Data? - After login, if there is some existing data, then...

**Comment by chrisguindon** [comment]
@flora8984461 Let's move your rough todo to issue #12

I would like to use this bug to discuss high-level topics with everyone involved in this project.

Once that is done, let's discuss how we can split your list into smaller issues (deliverables). 

For example, your next priority is to implement the HTML and CSS in order to recreate the look and feel that was proposed by @shanda-eclipse and @meKokabi.

Login is a separate issue/problem and  should not a blocker for updating the HTML/CSS,

Basically, I would like you to go through the same exercise that @autumnfound just did for the back-end! 

Feel free to reach out to me, I can help you figure out which bug we should create for the front-end!

**Comment by chrisguindon** [comment]
Just to update everyone. 

I created 4 projects to help us track requirements for this project:
https://github.com/EclipseFdn/react-eclipsefdn-members/projects

- Membership Enrollment: Website: Issues that are related to the website (front-end)
- Membership Enrollment: Infrastructure: Issues related to documentation, hosting and GDPR
- Membership Enrollment: REST API: Issues related to the REST API (back-end)
- Membership Portal: Empty for now but I plan to use it to start planing stage 2 of this project: Membership Portal.

**Comment by flora8984461** [comment]
@shanda-eclipse @meKokabi Hi Shanda and Mehrnaz, would you mind providing me the Adobe XD file? Since there are some assets I hope to get, like the icon on each step box, and the icon on the login page. 

Thanks!

**Comment by meKokabi** [comment]
Absolutely! I just need to double check the latest updates with Shanda and
will share it with you shortly after.

**Comment by flora8984461** [comment]
@meKokabi @shanda-eclipse  Thanks! Please feel free to check the preview on https://deploy-preview-21--eclipsefdn-react-members.netlify.app/form 

I wonder if this "remove this working group" button looks nice on Working Group step? (You can refer to the following screenshort)

![image](https://user-images.githubusercontent.com/39588094/106620571-b0e3db80-653f-11eb-8207-610a8570af11.png)

**Comment by meKokabi** [comment]
Here You are, link to Adobe XD: https://xd.adobe.com/view/583c3ff2-1fdc-43e4-ad8a-6dc730f5bb38-0d6f/

Also please note Shanda stressed on that the links (email at the end) should follow CSS of the rest of the website, and the header image is the same as the rest of the site, so when we change the homepage, this updates as well

**Comment by meKokabi** [comment]
[Uploading Membership-form (1) .xd.zip…]()

**Comment by chrisguindon** [comment]
> Here You are, link to Adobe XD: https://xd.adobe.com/view/583c3ff2-1fdc-43e4-ad8a-6dc730f5bb38-0d6f/
> 
> Also please note Shanda stressed on that all the links (email at the end) should follow CSS of the rest of the website, and the header image is the same as the rest of the site, so when we change the homepage, this updates as well

We don't see the email chain when you reply using email. I think it's best when you reply by login to Github.

Our plan is to use our default header/footer, however, we are not going to worry about this now. Our focus right now is on the content section.

**Comment by meKokabi** [comment]
Hi Chris, I have replied  and  embedded files on Github, please see the
screenshot. it is a zip folder.[image: Screen Shot 2021-02-02 at 11.14.17
AM.png]
Would you like me to add it again, in case there is any issue with opening
it?

Thanks for your support,

**Comment by chrisguindon** [comment]
@flora8984461 is this everything you need?

**Comment by chrisguindon** [comment]
Github does not allow us to move comments. I deleted this conversation and copied everyone comments in the correct location:
https://github.com/EclipseFdn/react-eclipsefdn-members/issues/17#issuecomment-771761504

This is a big project! Let's do our best to re-group conversation where it makes sense otherwise we are going to lose track of the requirements.

**Comment by meKokabi** [comment]
I converted XD files to PSD. Here You are!
[Membership-form (1) -xd.psd.zip](https://github.com/EclipseFdn/react-eclipsefdn-members/files/5912954/Membership-form.1.-xd.psd.zip)
I will also make jpg of each page and share it with you, just in case

**Comment by meKokabi** [comment]
<img width="646" alt="7" src="https://user-images.githubusercontent.com/75500459/106638203-9e729d80-6551-11eb-93b2-e559cbd2f557.png">
<img width="918" alt="1" src="https://user-images.githubusercontent.com/75500459/106638204-9e729d80-6551-11eb-871f-218a72eed8f8.png">
<img width="646" alt="4" src="https://user-images.githubusercontent.com/75500459/106638205-9f0b3400-6551-11eb-9707-d6e56ebaa651.png">
<img width="644" alt="5" src="https://user-images.githubusercontent.com/75500459/106638207-9f0b3400-6551-11eb-8828-5679c411dbec.png">
<img width="643" alt="6" src="https://user-images.githubusercontent.com/75500459/106638208-9f0b3400-6551-11eb-92ed-23eb6c78101f.png">
<img width="643" alt="2" src="https://user-images.githubusercontent.com/75500459/106638209-9fa3ca80-6551-11eb-976c-e46f53cdd8ba.png">
<img width="640" alt="3" src="https://user-images.githubusercontent.com/75500459/106638211-9fa3ca80-6551-11eb-8f3a-3f0c17687c8f.png">

**Comment by chrisguindon** [comment]
For Q1, we had a mission (MBO) to create a prototype, and I now ready to declare victory on that task:
https://eclipsefdn-react-members.netlify.app/

We will be more specific soon, but the next step for the dev team will be to switch from fake data to our custom REST API.

At the same time, we will need to look at user authentication.

**Comment by chrisguindon** [comment]
Closing this issue since it's too broad. Let's create issues if we need to discuss anything specific.
